bootstrap_salt/contrib/srv/salt/_grains/tags_to_grains.py

```diff
@@ -23,12 +23,27 @@ def get_aws_metadata():
         return {'custom_grain_error': True}
 
 
+_tags_to_grins_stack_name = 'NOT_FETCHED_YET'
+
+
+def get_stack_name():
+    global _tags_to_grins_stack_name
```

Comment:
Lul. Yes - mainly also to minimize the number of errors we see on the console output otherwise.